### DISCUSSION TITLE
Named hostile mobs protected in claims

### DISCUSSION THREAD

**Discussion by mdogdev** [GriefPrevention/GriefPrevention#1494] [Ideas]
As a player and server owner, there are constantly people that are killing named hostile mobs, like zombie pig-men, in farms. This is annoying as you have to have lots of support tickets open, and it annoys the mods who are constantly re-spawning the mobs.
If this feature were to be implemented, I think it would help a lot of people across multiple servers.

**Comment by Jikoo** [comment]
Sounds like you want [GPExtraProtections](https://github.com/GPAddons/GPExtraProtections).

**Reply by mdogdev** [reply]
Great, thanks!
